apps/communications/contacts/js/views/form.js

```diff
@@ -369,6 +371,36 @@ contacts.Form = (function() {
     };
   };
 
+  function showForm(event) {
+    if(event.key === 'Control') {
```

Comment:
And one more point, in all cases when you know that some functionality is not complete or will require additional changes please leave ``//TODO: `` comments describing the required changes and conditions when these changes shall be made